projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -40,6 +40,12 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   orders: {
     paths: ['my-account/orders'],
   },
+
+  // configuration related routes
+  configureCPQCONFIGURATOR: {
+    paths: ['configureCPQCONFIGURATOR/:rootProduct'],
+    paramsMapping: { rootProduct: 'rootProduct' },
```

Comment:
No need for explicit mapping. It's only useful when those names differ:
- the route `:param`
- the `params`' property passed with `cxRoute` to build a semantic link

Reply:
I removed paramsMapping